### PR TITLE
Add the form-builder as route

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: caluma-demo/app/router.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  location: config.locationType,
+  rootURL: config.rootURL
+});
+
+Router.map(function() {
+  this.mount("ember-caluma", {
+    as: "form-builder",
+    path: "/form-builder"
+  });
+  this.route('form-builder');
```
Comment (anehx) [inline caluma-demo/app/router.js]: This is a duplicated route

File: caluma-demo/app/routes/form-builder.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+});
```
Comment (anehx) [inline caluma-demo/app/routes/form-builder.js]: This file should not be needed

File: caluma-demo/app/styles/app.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$modal-z-index: 1;
+@import "ember-uikit";
+@import "ember-caluma";
```
Comment (anehx) [inline caluma-demo/app/styles/app.css]: This belongs into `app.scss`

File: caluma-demo/app/styles/app.scss [hunk 4]
```diff
@@ -0,0 +1,2 @@
+@import 'ember-uikit';
+@import 'ember-uikit';
```
Comment (anehx) [inline caluma-demo/app/styles/app.scss]: Duplicated

File: caluma-demo/app/templates/form-builder.hbs [hunk 5]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (anehx) [inline caluma-demo/app/templates/form-builder.hbs]: This file should not be needed

File: caluma-demo/config/targets.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  'last 1 Firefox versions',
+  'last 1 Safari versions'
+];
+
+const isCI = !!process.env.CI;
+const isProduction = process.env.EMBER_ENV === 'production';
+
+if (isCI || isProduction) {
+  browsers.push('ie 11');
+}
```
Comment (anehx) [inline caluma-demo/config/targets.js]: Let's not support IE11

File: caluma-demo/tests/unit/routes/form-builder-test.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | form-builder', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    let route = this.owner.lookup('route:form-builder');
+    assert.ok(route);
+  });
+});
```
Comment (anehx) [inline caluma-demo/tests/unit/routes/form-builder-test.js]: Unnecessary route

File: caluma-demo/translations/en-us.yaml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+      empty: You dont' have any petitions yet.
+      name: Name
+      slug: Slug
+      description: Description
+      isArchived: Archived
+      isPublished: Published
+    notification:
+      form:
+        create:
+          error: Error
```
Comment (anehx) [inline caluma-demo/translations/en-us.yaml]: Why are you overriding calumas already existing translations? Just use `en` as locale instead of `en-us`

**Review by anehx — CHANGES_REQUESTED** [review]
General:

* Use prettier :heart: 	
* Use yarn :sparkles:

File: caluma-demo/app/router.js (hunk 1 above)
Comment (JohnL17) [reply caluma-demo/app/router.js]: I have noticed this myself and have already deleted the route.

File: caluma-demo/app/routes/form-builder.js (hunk 2 above)
Comment (JohnL17) [reply caluma-demo/app/routes/form-builder.js]: I have deleted the route form-builder.

File: caluma-demo/app/styles/app.scss (hunk 4 above)
Comment (JohnL17) [reply caluma-demo/app/styles/app.scss]: I have deleted the duplicated line.

File: caluma-demo/app/styles/app.css (hunk 3 above)
Comment (JohnL17) [reply caluma-demo/app/styles/app.css]: Changed it.

File: caluma-demo/app/templates/form-builder.hbs (hunk 5 above)
Comment (JohnL17) [reply caluma-demo/app/templates/form-builder.hbs]: This was generated automatically when I created the route form-builder. Removed it.

File: caluma-demo/config/targets.js (hunk 6 above)
Comment (JohnL17) [reply caluma-demo/config/targets.js]: Ok I have deleted it.

File: caluma-demo/tests/unit/routes/form-builder-test.js (hunk 7 above)
Comment (JohnL17) [reply caluma-demo/tests/unit/routes/form-builder-test.js]: I changed this already.

File: caluma-demo/translations/en-us.yaml (hunk 8 above)
Comment (JohnL17) [reply caluma-demo/translations/en-us.yaml]: Oops didn't know that was how it worked. Will change it.

File: caluma-demo/app/styles/app.css (hunk 3 above)
Comment (anehx) [inline caluma-demo/app/styles/app.css]: You can delete this file

File: caluma-demo/app/styles/app.scss [hunk 9]
```diff
@@ -0,0 +1,12 @@
+$global-primary-background: #3799ad;
+
+$global-font-family: "Oxygen", Helvetica, Arial, sans-serif;
+$base-code-font-family: "Oxygen Mono", monospace;
```
Comment (anehx) [inline caluma-demo/app/styles/app.scss]: Use the "Ubuntu" font

File: caluma-demo/config/targets.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const browsers = [
+  "last 1 Chrome versions",
+  "last 1 Firefox versions",
+  "last 1 Safari versions"
+];
+
+const isCI = !!process.env.CI;
+const isProduction = process.env.EMBER_ENV === "production";
```
Comment (anehx) [inline caluma-demo/config/targets.js]: Those two lines are not needed anymore

File: caluma-demo/package.json [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+    "graphql": "^14.0.2",
+    "loader.js": "^4.7.0",
+    "prettier": "^1.18.2",
+    "qunit-dom": "^0.8.4",
+    "sass": "^1.22.10"
+  },
+  "engines": {
+    "node": "8.* || >= 10.*"
+  },
+  "dependencies": {}
```
Comment (anehx) [inline caluma-demo/package.json]: Empty, not needed

File: caluma-demo/translations/en.yaml [hunk 12]
```diff
@@ -0,0 +1 @@
+price_banner: The {product} costs {price, number, USD}
```
Comment (anehx) [inline caluma-demo/translations/en.yaml]: Why this?

File: caluma-demo/package.json [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+    "ember-math-helpers": "^2.11.3",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-pikaday": "^2.4.0",
+    "ember-power-select": "^3.0.3",
+    "ember-promise-helpers": "^1.0.9",
+    "ember-qunit": "^4.4.1",
+    "ember-resolver": "^5.0.1",
+    "ember-source": "~3.12.0",
+    "ember-uikit": "^0.8.2",
+    "ember-welcome-page": "^4.0.0",
```
Comment (anehx) [inline caluma-demo/package.json]: Delete `ember-welcome-page`

File: caluma-demo/package.json [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+    "ember-engines": "^0.8.2",
+    "ember-export-application-global": "^2.0.0",
+    "ember-fetch": "^6.3.1",
+    "ember-intl": "^4.1.0",
+    "ember-load-initializers": "^2.0.0",
+    "ember-math-helpers": "^2.11.3",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-pikaday": "^2.4.0",
+    "ember-power-select": "^3.0.3",
+    "ember-promise-helpers": "^1.0.9",
```
Comment (anehx) [inline caluma-demo/package.json]: Delete `ember-promise-helpers`

File: caluma-demo/package.json [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+    "ember-cli-htmlbars": "^3.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-moment-shim": "^3.7.1",
+    "ember-cli-sass": "^10.0.1",
+    "ember-cli-showdown": "^4.4.4",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-template-lint": "^1.0.0-beta.1",
+    "ember-cli-uglify": "^2.1.0",
+    "ember-data": "~3.12.0",
```
Comment (anehx) [inline caluma-demo/package.json]: Delete `ember-data`

File: caluma-demo/package.json [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+    "ember-cli-app-version": "^3.2.0",
+    "ember-cli-babel": "^7.7.3",
+    "ember-cli-dependency-checker": "^3.1.0",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars": "^3.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-moment-shim": "^3.7.1",
+    "ember-cli-sass": "^10.0.1",
+    "ember-cli-showdown": "^4.4.4",
```
Comment (anehx) [inline caluma-demo/package.json]: Delete `ember-cli-showdown`

File: caluma-demo/config/optional-features.json [hunk 17]
```diff
@@ -0,0 +1,3 @@
+{
+  "jquery-integration": true
```
Comment (anehx) [inline caluma-demo/config/optional-features.json]: Add `"application-template-wrapper": false`

File: caluma-demo/app/services/apollo.js [hunk 18]
```diff
@@ -0,0 +1,4 @@
+import ApolloService from "ember-apollo-client/services/apollo";
+import CalumaApolloServiceMixin from "ember-caluma/mixins/caluma-apollo-service-mixin";
+
+export default ApolloService.extend(CalumaApolloServiceMixin, {});
```
Comment (anehx) [inline caluma-demo/app/services/apollo.js]:
```suggestion
export default class CustomApolloService extends ApolloService.extend(CalumaApolloServiceMixin) {};
```

File: caluma-demo/app/routes/application.js [hunk 19]
```diff
@@ -0,0 +1,10 @@
+import { inject as service } from "@ember/service";
+import Route from "@ember/routing/route";
+
+export default Route.extend({
+  intl: service(),
+  beforeModel() {
+    this._super(...arguments);
+    return this.intl.setLocale("en");
+  }
+});
```
Comment (anehx) [inline caluma-demo/app/routes/application.js]:
```js
export default class ApplicationRoute extends Route {
  @service intl;

  beforeModel() {
    super.beforeModels(...arguments);

    return this.intl.setLocale("en");
  }
};
```

File: caluma-demo/app/routes/about.js [hunk 20]
```diff
@@ -0,0 +1,3 @@
+import Route from "@ember/routing/route";
+
+export default Route.extend({});
```
Comment (anehx) [inline caluma-demo/app/routes/about.js]:
```suggestion
export default AboutRoute extends Route {}
```

File: caluma-demo/.travis.yml [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+
+env:
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+script:
+  - npm run lint:hbs
+  - npm run lint:js
+  - npm test
```
Comment (anehx) [inline caluma-demo/.travis.yml]: Yarn for CI as well:

```yml
---
language: node_js
node_js:
  - "8"

sudo: false
dist: trusty

addons:
  chrome: stable

cache:
  yarn: true

env:
  global:
    # See https://git.io/vdao3 for details.
    - JOBS=1

before_install:
  - curl -o- -L https://yarnpkg.com/install.sh | bash
  - export PATH=$HOME/.yarn/bin:$PATH

install:
  - yarn install --non-interactive

script:
  - yarn lint:hbs
  - yarn lint:js
  - yarn test
```

**Review by anehx — CHANGES_REQUESTED** [review]

**Review by anehx — APPROVED** [review]

**Pull request merged**